include/glow/Base/Tensor.h

```diff
@@ -364,6 +364,24 @@ class Tensor final {
     }
   }
 
+  /// Cast each element of this tensor to \p newTy.
+  void cast(ElemKind newTy) {
```

Comment:
Maybe a better name would be convertToType?

Reply:
Sounds gods!